geomesa-kafka/geomesa-kafka-confluent/src/main/scala/org/locationtech/geomesa/kafka/confluent/ConfluentKafkaDataStoreFactory.scala

```diff
@@ -54,14 +62,47 @@ object ConfluentKafkaDataStoreFactory extends GeoMesaDataStoreInfo with LazyLogg
   override val DisplayName = "Confluent Kafka (GeoMesa)"
   override val Description = "Confluent Apache Kafka\u2122 distributed log"
 
-  val SchemaRegistryUrl = new GeoMesaParam[URL]("kafka.schema.registry.url", "URL to a confluent schema registry server, used to read Confluent schemas (experimental)")
+  val SchemaRegistryUrl =
+    new GeoMesaParam[URL](
+      "kafka.schema.registry.url",
+      "URL to a confluent schema registry server, used to read Confluent schemas (experimental)",
+      optional = false
+    )
+
+  val SchemaOverrides =
+    new GeoMesaParam[String](
+      "kafka.schema.overrides",
+      "Typesafe configuration defining a map from topic name to schema override (experimental)",
+      optional = true,
+      largeText = true
+    )
 
   override val ParameterInfo: Array[GeoMesaParam[_ <: AnyRef]] =
-    KafkaDataStoreFactory.ParameterInfo.+:(SchemaRegistryUrl)
+    KafkaDataStoreFactory.ParameterInfo ++ Array(SchemaRegistryUrl, SchemaOverrides)
```

Comment:
```suggestion
    Array(SchemaRegistryUrl) ++ KafkaDataStoreFactory.ParameterInfo ++ Array(SchemaOverrides)
```

This would probably make config easier, since the geoserver page orders the input properties according to this. Previously the `+:` was pre-pending the schema registry url, so this will keep that behavior and then add the SchemaOverrides below required things like kafka brokers, etc.